test/DXILValidation/lit.local.cfg

```diff
@@ -0,0 +1,3 @@
+# Enable .ll suffixes for this directory
+# They are disabled above to avoid tests unsupported by HLSL
+config.suffixes = ['.txt', '.td', '.test', '.ll']
```

Comment:
why .td here?

Reply:
Just matching what's already in the upper directory's `test/lit.cfg` since the intent here is only to add .ll rather than change anything else

Reply:
If we only care .ll in this directory, maybe config.suffixes = ['.ll'] is what we want.